geoportailv3/static/js/mymaps/mymaps.html

```diff
@@ -1,11 +1,63 @@
-<div><!-- TODO block to be shown only in cases where no map is selected -->
+<div ng-show="!ctrl.mymap">
   <p translate>
     Create, save and share your own maps.
   </p>
-  <p class="text-center" ng-if="mainCtrl.loggedin">
-    <button class="btn btn-default" translate>Open a map</button>
+  <p class="text-center"><!-- Show only when logged in -->
+    <button class="btn btn-default" ng-click="ctrl.choose()" translate>Open a map</button>
   </p>
   <p class="text-center">
-    <button class="btn btn-default" translate>Create new map</button>
+    <button class="btn btn-default" ng-click="ctrl.create()" translate>Create new map</button>
   </p>
 </div>
+
+<!-- Map displayed -->
+<div ng-show="ctrl.mymap && ctrl.mymap.id">
+  <div class="panel-header">
+    <button type="button" class="close" aria-label="Close" ng-click="ctrl.close()"><span aria-hidden="true">&times;</span></button>
+  </div>
+  <p class="title" ng-click="ctrl.modify()">{{ctrl.mymap.title}}</p>
+  <p class="description" ng-click="ctrl.modify()" ng-if="ctrl.mymap.description">{{ctrl.mymap.description}}</p>
+  <p class="no-description" ng-click="ctrl.modify()" ng-if="!ctrl.mymap.description" translate><em>No description</em></p>
+</div>
+
+<!-- Modification Modal -->
+<ngeo-modal ngeo-modal-shown="ctrl.modifying">
+  <div class="modal-header">
+    <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+    <h4 class="modal-title" translate>Modify title and description of the map</h4>
+  </div>
+  <div class="modal-body">
+    <form>
+    <div class="form-group">
+      <label for="mymapsTitle" translate>Title of the map</label>
+      <input type="text" class="form-control" id="mymapsTitle" ng-model="ctrl.newTitle">
```

Comment:
OK for removing the ids. However, I moving the `input` inside the label has side effects in bootstrap.